src/modules/frontend/outbound/person.service.ts

```diff
@@ -7,11 +7,20 @@ import { CreatePersonBodyParams } from '../../person/api/create-person.body.para
 import { PersonenQueryParams } from '../../person/api/personen-query.param.js';
 import { User } from '../auth/user.decorator.js';
 import { PaginatedResponseDto } from '../api/paginated-data.response.js';
+import { PersonByIdParams } from '../../person/api/person-by-id.param.js';
 
 @Injectable()
 export class PersonService {
     public constructor(private httpService: BackendHttpService) {}
 
+    public async getPersonById(params: PersonByIdParams): Promise<PersonendatensatzResponse> {
```

Comment:
Also pass in User for future auth (see `getAllPersons`)

Reply:
(y)